docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.5'
+
+services:
+  rack:
+    build: .
+    ports: 
+      - 9292:9292
```

Comment:
It is a good practice to specify container name
Volumes would be good to have bundle folder, so that if you change your gemfile, you don't need to rebuild whole container. Just run `docker compose exec rack bundle install` and that's it

```yml
services:
  rack:
    build: .
    container_name: rack
    volumes:    
      - ./bundle:/opt/bundle
```

Reply:
fixed